routers/api/v1/api.go

```diff
@@ -955,16 +959,16 @@ func Routes() *web.Router {
 
 		// Users (requires user scope)
 		m.Group("/users", func() {
-			m.Get("/search", reqExploreSignIn(), user.Search)
+			m.Get("/search", reqExploreSignInAndUsersExploreEnabled(), user.Search)
 
 			m.Group("/{username}", func() {
-				m.Get("", reqExploreSignIn(), user.GetInfo)
+				m.Get("", reqExploreSignInAndUsersExploreEnabled(), user.GetInfo)
```

Comment:
The logic here is strange. If explore is disabled, and user can not get user info?

Reply:
I see. It should only works on `/users/search`. I've fixed it in 7314a2788fd302873ad61c8cae58721fb529a4bc